playwright-tests/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "skipLibCheck": true,
+    "alwaysStrict": true,
+    "strict": true,
+    "esModuleInterop": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "strictNullChecks": false,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": false,
+    "declaration": true,
+    "declarationMap": true,
+    "incremental": true,
+    "forceConsistentCasingInFileNames": true,
+    "moduleResolution": "node",
+    "module": "commonjs",
+    "target": "ES2020",
+    "resolveJsonModule": true,
+    "composite": true,
+    "sourceMap": true,
+    "lib": [
+      "ES2020",
+      "dom"
+    ],
+    "baseUrl": ".",
+    "paths": {
+      "@testData": ["tests/widget/testData"],
```

Comment:
I guess we can remove allias paths

Reply:
I forget to change it for a new project
I think the better I will fix it instead of removing